fMRISurface/GenericfMRISurfaceProcessingPipeline.sh

```diff
@@ -105,7 +105,7 @@ esac
 
 if [ "${RegName}" = "FS" ] ; then
     log_Warn "WARNING: FreeSurfer's surface registration (based on cortical folding) is deprecated in the"
-    log_Warn "         HCP Pipelines as it results in poorer cross-subject functional and cortical areal "
+    log_Warn "         HCP Pipelines as it results in poorer cross-session functional and cortical areal "
```

Comment:
This probably shouldn't have been find/replaced here.

Reply:
Agree